eyeshade/workers/publishers.js

```diff
@@ -25,16 +27,23 @@ exports.workers = {
       const {
         shouldUpdateBalances,
         settlementId,
+        publisher,
         type
       } = payload
 
-      const docs = await settlementsCollection.find({
+      const query = {
         type,
         settlementId,
         owner: {
           $exists: true
         }
-      })
+      }
+
+      if (publisher) {
```

Comment:
i don't see publisher passed from anywhere. is this supposed to be triggered from an external service?

Reply:
yeah this is e.g. if we want to run a manual script that enters in specific settlement transactions that were missed in the original aggregate job